dept44-scheduler/src/test/java/se/sundsvall/dept44/scheduling/Dept44SchedulerAspectTest.java

```diff
@@ -0,0 +1,88 @@
+package se.sundsvall.dept44.scheduling;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import se.sundsvall.dept44.scheduling.health.Dept44CompositeHealthContributor;
+import se.sundsvall.dept44.scheduling.health.Dept44HealthIndicator;
+
+class Dept44SchedulerAspectTest {
+
+	private Dept44SchedulerAspect aspect;
+
+	@Mock
+	private ProceedingJoinPoint pjp;
+
+	@Mock
+	private Dept44CompositeHealthContributor healthContributor;
+
+	@Mock
+	private Dept44Scheduled dept44Scheduled;
+
+	@BeforeEach
+	void setUp() {
+		MockitoAnnotations.openMocks(this);
```

Comment:
Is there any advantage using this call, instead of: the more common
`@ExtendWith(MockitoExtension.class)` ??

Reply:
Not really. Will change